roles/ssh/files/sshd_config

```diff
@@ -0,0 +1,45 @@
+# running from inetd
+# Port 2200
+Protocol 2
+HostKey /etc/ssh/ssh_host_rsa_key
+HostKey /etc/ssh/ssh_host_dsa_key
+
+UsePrivilegeSeparation no
+
+SyslogFacility auth
+LogLevel info
+
+PermitRootLogin yes
+
+PrintMotd yes
```

Comment:
Hmm.  I wonder if this might be messing with jenkins?  Seems unlikely but I'm grasping at straws.